taxi_service/settings.py

```diff
@@ -79,6 +80,7 @@
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
         "NAME": BASE_DIR / "db.sqlite3",
+        "DIRS": [BASE_DIR / "templates"],
```

Comment:
'DIRS' is not a valid setting for the DATABASES 'default' option. Please remove this line.